docs/reference/compatibility.txt

```diff
@@ -91,13 +95,25 @@ is deprecated.
      - |checkmark|
      - |checkmark|
      - |checkmark|
-     - |checkmark|
+     -
      -
      -
      -
      -
      - |checkmark|
 
+   * - 7.5
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - D
+     -
+     -
+     -
+     - |checkmark|
+     -
```

Comment:
@p-mongo we have a problem, if we deprecate JRuby-9.2 in 7.5, there will be no available version of JRuby, since we don't support JRuby-9.3 until 8.0